index.js

```diff
@@ -2,23 +2,30 @@
 'use strict';
 
 var VersionChecker = require('ember-cli-version-checker');
+var path = require('path');
 
 module.exports = {
   name: 'ember-resolver',
 
-  init: function() {
-    this._super.init.apply(this, arguments);
-    this.options = this.options || {};
-
+  emberResolverFeatureFlags() {
     var config = this.project.config();
     var resolverConfig = config['ember-resolver'] || {};
 
-    var resolverFeatureFlags = Object.assign({
+    return Object.assign({
       /* Add default feature flags here */
+      EMBER_RESOLVER_MODULE_UNIFICATION: true
```

Comment:
This line must be changed before merge. It defaults the FF to `true` when what we want is: `true` only if configured or when running tests.